### ISSUE TITLE
Parameter setting problem

### ISSUE THREAD

**Issue by hopewolf** [donydchen/ganimation_replicate#6]
I notice the parameters d_cond=160 and loss_mask=0 loss_mask_smooth =0 in your program but I find it's d_cond=4000 and loss_mask=0.1 loss_mask_smooth=1e-4 in original paper, Could you please tell me how can  you set these parameters?Thank you

**Comment by donydchen** [comment]
Let's say the using of the attention mask and the color transformation maybe just some paper tricks. Kindly refer to https://github.com/albertpumarola/GANimation/issues/21, at the time the paper just came out, the implementation of Eq. (1) is different from the paper. You can verify it by differing the v1 and the v2 of https://arxiv.org/abs/1807.09251 
You see, even by swapping the attention and the color mask, it still works. So I guess maybe this part is not so important, that is why I set these two parameters to 0. 
Kindly note that those are just my guess, so please use it at your own risk if applicable.

**Comment by hopewolf** [comment]
Thans for your reply.When I experiment in your code and Ganimation, the attention mask behaves diffierent  in two codes.
When I do experiment in your replicate code,the mean value of attention mask is 0.96-0.98 in first 3 epochs,but when I do experiment in the original Ganimaition,the attention mask is saturated to 1 in first epoch with the same parameters in your code and never change to other value.
Moreover, I do some experiment  that setting lambda_mask to 3,4,5 in Ganimation and it really prevent the attention mask from goint to 1 and sometimes down to 0.
Could you please tell me how could  you handle these quesitions, because I can't find any thing diffierent,like parameters,training methods,or dataset(celeba), between these two codes.
Thanks in advance.

**Comment by donydchen** [comment]
Hi @hopewolf , I didn't run and test the original GANimation project before, instead I replicate it from the very beginning. So I'm not familiar with details of the original project. 
You may try to replicate GANimation by yourself, hopefully through which you will find the answer to your doubt.

**Comment by hopewolf** [comment]
@donydchen Thanks for your answer, I'll check with that, your answer is really helpful.
